### ISSUE TITLE
Define Default Borders (e.g. in Buttons) & Radii

### ISSUE THREAD

**Issue by annatuma** [cc-archive/vocabulary-legacy#39]
**Description**
We need to define what our default borders are, for example, for buttons, and document that in Figma DL.

We also need to review each of the current Radii components (three layers: Rem, Em, Special), to determine which ones we want to keep versus remove at this time. **We must keep the necessary radius for buttons, based on our current button design.** There may be other use cases.

This will require some discussion, which should happen in this ticket.

**Comment by annatuma** [comment]
@brenoferreira and @hugosolar please review the Radii components in Vocabulary and think about which ones we need to keep and which ones we can remove:https://cc-vue-vocabulary.netlify.com/storybook/?path=/story/tokens-radii--rem-based

Please indicate your views here.

@panchovm will define our default borders for buttons in these terms, and think about other uses cases.

**Comment by hugosolar** [comment]
I'm in favor of using just 1 radio value, and that will be the button radius. I don't think we need more than that. 
Also, I think we should use just `rem` units (but maybe that's part of another discussion)

**Comment by fcoveram** [comment]
In the case of Figma, it is not allowed to keep borders blocked. That means, no matter our definition, it will be only documented as text instead of blocked by component (i.e., buttons and entries)

**Comment by kgodey** [comment]
I don't think we need radii since we're styling each element on its own. If we do, we can create a new issue.
